freqtrade/data/entryexitanalysis.py

```diff
@@ -0,0 +1,259 @@
+import logging
+import os
+from pathlib import Path
+from typing import List, Optional
+
+import joblib
+import pandas as pd
+from tabulate import tabulate
+
+from freqtrade.data.btanalysis import get_latest_backtest_filename, load_backtest_data
+from freqtrade.exceptions import OperationalException
+
+
+logger = logging.getLogger(__name__)
+
+
+def _load_signal_candles(backtest_dir: Path):
+
+    if backtest_dir.is_dir():
+        scpf = Path(backtest_dir,
+                    os.path.splitext(
+                        get_latest_backtest_filename(backtest_dir))[0] + "_signals.pkl"
+                    )
+    else:
+        scpf = Path(os.path.splitext(
+                        get_latest_backtest_filename(backtest_dir))[0] + "_signals.pkl"
+                    )
+
+    try:
+        scp = open(scpf, "rb")
+        signal_candles = joblib.load(scp)
+        logger.info(f"Loaded signal candles: {str(scpf)}")
+    except Exception as e:
+        logger.error("Cannot load signal candles from pickled results: ", e)
+
+    return signal_candles
+
+
+def _process_candles_and_indicators(pairlist, strategy_name, trades, signal_candles):
+    analysed_trades_dict = {}
+    analysed_trades_dict[strategy_name] = {}
+
+    try:
+        logger.info(f"Processing {strategy_name} : {len(pairlist)} pairs")
+
+        for pair in pairlist:
+            if pair in signal_candles[strategy_name]:
+                analysed_trades_dict[strategy_name][pair] = _analyze_candles_and_indicators(
+                                                              pair,
+                                                              trades,
+                                                              signal_candles[strategy_name][pair])
+    except Exception:
+        pass
```

Comment:
I don't think we should swallow all exceptions blindly.
if we expect certain exceptions (in my case it's a keyerror due to a missing `-s` argument) - then it should also be shown unless we expected that error beforehand.

Reply:
no push? (intentional maybe?) ... ;) just saying :laughing:

Reply:
Done